packages/teleterm/src/ui/services/clusters/clustersService.ts

```diff
@@ -413,6 +413,10 @@ export class ClustersService extends ImmutableStore<ClustersServiceState> {
     return cluster.loggedInUser?.assumedRequests || {};
   }
 
+  getAssumedRequest(clusterUri: string, requestId: string) {
```

Comment:
@avatus Do methods such as `getAssumedRequests`, `getRequestableRoles`, `getAccessRequests` and so on work only with root clusters? If so, it'd be worthwhile to update the argument name to reflect that.

Reply:
I believe these do not rely on root cluster as they either defer to the access request id (from whatever connected auth server) and then role mapping for the others so we should be good here.

Reply:
I'm afraid I don't understand, are we good here to change the arg name or do those methods need to work with leaf clusters too, so the arg name should stay as `clusterUri`? 😶

I just looked into this and:

* `getAssumedRequests` does `cluster.loggedInUser?.assumedRequests`. `assumedRequests` is present only for root clusters so here we should definitely change the arg name to reflect that.
* `getRequestableRoles` and `getAccessRequests` reach out to the auth server. So I think it's for root clusters only?

I think the overarching question is: could the user ever perform any operations on access requests on a leaf cluster? If not, then all methods in `ClustersService` that accept `clusterUri` as an arg should have that arg renamed to `rootClusterUri` IMHO.

Reply:
I think we are saying the same thing but from two different perspectives. I'm saying "it all goes to the root cluster auth server anyway so it doesn't matter what clusterUri is sent (root or leaf)" and you're saying "because it goes to the root cluster auth server, we should name that variable to rootClusterUri so they know". Now that I understand what you're saying, I agree with your perspective more.